### ISSUE TITLE
AVT1: The combobox design pattern is detected as WAI-ARIA 1.1, which is not allowed by WAI-ARIA 1.2

### ISSUE THREAD

**Issue by icemanlily** [carbon-design-system/carbon#7992]
===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: IBM 4.1.2 Name, Role, Value W3C WAI-ARIA v1.2 Authoring Practices - Combobox
IBM Guidance: https://www.w3.org/TR/wai-aria-1.2/#combobox
Test Tool Type: IBM Equal Access Accessibility Checker
===== EXPECTED RESULT =====
Pls check the followed link for detail:
IBM 4.1.2 Name, Role, Value W3C WAI-ARIA v1.2 Authoring Practices - Combobox :
https://www.w3.org/TR/wai-aria-1.2/#combobox
![image](https://user-images.githubusercontent.com/6970471/110060049-43c2a080-7da0-11eb-9e64-c2c6976e0276.png)

**Comment by tw15egan** [comment]
@icemanlily can you confirm this is still present on the latest version (`v10.30.0`)? We just pushed [an update](https://github.com/carbon-design-system/carbon/pull/7777) to comply with WAI-ARIA 1.2

**Comment by icemanlily** [comment]
@tw15egan OK. We will upgrade the Carbon version and re-test it. Thank you!
